src/pages/edge/edge-network/tabs/api-reference.md

```diff
@@ -329,6 +514,61 @@ NSDictionary *data = @{ @"sample" : @"data"};
 }];
 ```
 
+##### Example with Datastream ID override
+
+```objectivec
+// Create Experience event from dictionary:
+NSDictionary *xdmData = @{ @"eventType" : @"SampleXDMEvent"};
+NSDictionary *data = @{ @"sample" : @"data"};
+AEPExperienceEvent* event = [[AEPExperienceEvent alloc]initWithXdm:xdmData data:data datastreamIdOverride: @"SampleDatastreamIdOverride"];
+```
+
+```objectivec
```

Comment:
nit: let's club these two blocks in a single one - same below
also variant count will need to be updated

Reply:
can you update the next example as well and update the variant repeat count for both